pom.xml

```diff
@@ -1188,6 +1188,8 @@
 
     <module>arquillian</module>
 
+    <module>fractions/health-api</module>
```

Comment:
As this isn't a fraction, I'm thinking we need to add a top level directory `apis`, or the like, to add this there, and also move `jaxrs-client-api` there as well.